codalab/rest/interpret.py

```diff
@@ -693,7 +693,7 @@ def resolve_items_into_infos(items):
         )
         if bundle_info is not None:
             bundle_info['sort_key'] = i['sort_key']
-            bundle_info['id'] = i['id']
+            bundle_info = dict(bundle_info, id=i['id'])
```

Comment:
Should we remove the earlier line and instead do:

```suggestion
            bundle_info = dict(bundle_info, id=i['id'], sort_key=i['sort_key'])
```